### PR TITLE
feature(toolbar): Add text complementing toolbar

### PR DESCRIPTION
**Pull request by jdetle**
This is a very small fix after seeing an improvement from some changes I think @peggyrayzis made in her cell-creator refactor. On hover, text now appears explaining the actions corresponding to each icon in the toolbar.

### CODE REVIEW

File: src/notebook/components/cell/toolbar.js [hunk 1]
```diff
@@ -93,19 +93,31 @@ export default class Toolbar extends React.PureComponent {
         <div className="cell-toolbar">
           {showPlay &&
           <span>
-            <button onClick={this.executeCell} className="executeButton" >
+            <button
+              onClick={this.executeCell}
+              title= "execute cell"
```
Comment (lgeiger) [inline src/notebook/components/cell/toolbar.js]: Looks like eslint isn't happy with the space behind `=`.

Comment (jdetle) [reply src/notebook/components/cell/toolbar.js]: Yeah I couldn't run the linter because a GeoJSON test is failing on my machine :\

Comment (jdetle) [reply src/notebook/components/cell/toolbar.js]: Trying npm run reinstall right now

Comment (lgeiger) [reply src/notebook/components/cell/toolbar.js]: GeoJSON tests should be fixed by #1401.

You can also run `npm run test:lint` if you only want linting or try https://atom.io/packages/linter-eslint

**Comment by jdetle** [comment]
Thanks for the catch @lgeiger. Made the fix.

**Review by lgeiger — APPROVED** [review]

**Pull request merged**